chainer/testing/matrix.py

```diff
@@ -0,0 +1,52 @@
+import numpy
+
+from chainer.utils import argument
+
+
+def generate_matrix(shape, dtype=float, **kwargs):
+    r"""Generates a random matrix with given singular values.
+
+    This function generates a random NumPy matrix (or a set of matrices) that
```

Comment:
It seems NumPy calls it "a stack of matrices".

Reply:
Thanks for the suggestion. Fixed to use NumPy terminology.